sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/DefaultWizard.java

```diff
@@ -779,7 +779,7 @@ private Map<String, String> getDistroVersionsOptionsMap(Set<String> versions, Ve
 	private Map<String, String> getO3VersionsOptionsMap(VersionsHelper versionsHelper,
 			String optionTemplate, String artifactTemplate) {
 		Map<String, String> optionsMap = new LinkedHashMap<>();
-		Artifact artifact = new Artifact("referenceapplication-distro", "3.0.0-SNAPSHOT", "org.openmrs.distro", "zip");
+		Artifact artifact = new Artifact("referenceapplication-distro", "version", "org.openmrs.distro", "zip");
```

Comment:
Does this work with the string `"version"`? I would've thought this should be `null` or something?